sklearn/ensemble/_hist_gradient_boosting/loss.py

```diff
@@ -141,6 +153,55 @@ def update_gradients_and_hessians(self, gradients, hessians, y_true,
         _update_gradients_least_squares(gradients, y_true, raw_predictions)
 
 
+class LeastAbsoluteDeviation(BaseLoss):
+    """Least asbolute deviation, for regression.
+
+    For a given sample x_i, the loss is defined as::
+
+        loss(x_i) = |y_true_i - raw_pred_i|
+    """
+
+    hessians_are_constant = True
+    need_update_leaves_values = True  # see note in BaseLoss
+
+    def __call__(self, y_true, raw_predictions, average=True):
+        # shape (1, n_samples) --> (n_samples,). reshape(-1) is more likely to
+        # return a view.
+        raw_predictions = raw_predictions.reshape(-1)
+        loss = np.abs(y_true - raw_predictions)
+        return loss.mean() if average else loss
+
+    def get_baseline_prediction(self, y_train, prediction_dim):
+        return np.median(y_train)
+
+    @staticmethod
+    def inverse_link_function(raw_predictions):
```

Comment:
I see that you add a test for this function for the least-square case. Would it make sense to have something similar here.